### PR TITLE
Include more information in the return from reconstruct().

### PR DESCRIPTION
**Pull request by efiring**
The returned Bunch now includes the kwargs and the times,
with the latter both in the original form and in matplotlib
datenums.

Added two convenience Bunch objects to the utide namespace:
hours_per_cycle and cycles_per_hour, both indexed with the
constituent name.

Updated the utide_real_data_example.ipynb notebook, and
stored it without output.

### CODE REVIEW

**Comment by efiring** [comment]
Does anyone know where the data file used in the notebook comes from?  It would be nice to say something about it--e.g., it is the record from Harbor X in country Y for 1998, from Z hydrographic service.

**Comment by efiring** [comment]
The failing check is just the coding standard, so the failure will go away when #54 is merged.

File: utide/__init__.py [hunk 1]
```diff
@@ -2,12 +2,15 @@
 
 from ._solve import solve
 from ._reconstruct import reconstruct
-from ._ut_constants import ut_constants, constit_index_dict
+from ._ut_constants import (ut_constants, constit_index_dict,
+    hours_per_cycle, cycles_per_hour)
```
Comment (wesleybowman) [inline utide/__init__.py]: Just a PEP8 thing, but shouldn't we align this with the `(` from above?

**Comment by ocefpaf** [comment]
:+1: to these improvements. Thanks @efiring!

**Comment by efiring** [comment]
I think that only @ocefpaf and @wesleybowman have commit rights, correct?  Would one of you merge this, please?

**Comment by ocefpaf** [comment]
Sure. I was waiting for @wesleybowman input but this one good to go IMO.

**Comment by ocefpaf** [comment]
> @ocefpaf I thought I had :sweat_smile: And I thought @efiring had permissions to merge as well. My bad.

So did I :smile:

**Pull request merged**